content/docs/intro/concepts/programming-model.md

```diff
@@ -2624,7 +2624,7 @@ Any outputs can be returned by your `create` function in the `outs` property of
 
 > **Note:** The following only applies to statically typed languages.
 
-If you need to access the outputs of your custom resource outside it with strong typing support, declare each output property returned in the `outs` property by your `create` function as a class member of the `pulumi.dynamic.Resource` itself. For example, in TypeScript, these must be declared as `public readonly` class members in your `pulumi.dynamic.Resource` class. These class members must also have the type `pulumi.Output<T>`.
+If you need to access the outputs of your custom resource outside it with strong typing support, declare each output property returned in the `outs` property by your `create` function as a class member of the `pulumi.dynamic.Resource` itself. For example, in TypeScript, these must be declared as `public readonly` class members in your `pulumi.dynamic.Resource` class. These class members must also have the type `pulumi.Output<T>`. In TypeScript, if `--strictPropertyInitialization` is enabled (it typically is enabled by default along with other `--strict` mode flags), [definite assignment assertion modifiers](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#strict-class-initialization) (an `!` exclamation point) must be used on each class member to relay to TypeScript that these will be initialized indirectly.
```

Comment:
What do you think of this?

```suggestion
If you need to access the outputs of your custom resource outside it with strong typing support, declare each output property returned in the `outs` property by your `create` function as a class member of the `pulumi.dynamic.Resource` itself.

For example, in TypeScript, these must be declared as `public readonly` class members in your `pulumi.dynamic.Resource` class. These class members must also have the type `pulumi.Output<T>`.

> If `--strictPropertyInitialization` is enabled (it typically is enabled by default along with other `--strict` mode flags), [definite assignment assertion modifiers](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#strict-class-initialization) (an `!` exclamation point) must be used on each class member to relay to TypeScript that these will be initialized indirectly.
```

Reply:
I considered doing it like this, but then the blockquote gets combined with the one directly below it, which doesn't look quite right:

<img width="743" alt="Screen Shot 2020-02-19 at 2 02 22 PM" src="https://user-images.githubusercontent.com/710598/74880631-75490500-5320-11ea-9696-36db83bad204.png">

Reply:
Ah gotcha. Could we then remove the blockquote for the "Note"?

Reply:
How about this: I moved the existing note into the first paragraph, and added this new blockquote as it's own note. What do you think?

<img width="741" alt="Screen Shot 2020-02-19 at 2 32 36 PM" src="https://user-images.githubusercontent.com/710598/74882988-e12d6c80-5324-11ea-8590-d32dd622820e.png">

Reply:
That looks good! Thank you, @justinvp.